src/solvers/dg/3d/dg.jl

```diff
@@ -1022,6 +1022,37 @@ function analyze_solution(dg::Dg3D, mesh::TreeMesh, time::Real, dt::Real, step::
     println()
   end
 
+  # L2/L∞ errors of the primitive variables
+  if :l2_error_primitive in dg.analysis_quantities || :linf_error_primitive in dg.analysis_quantities
+    l2_error, linf_error = calc_error_norms(cons2prim, dg, time)
```

Comment:
Won't this override the previously calculated L2/Linf errors on the conservative variables? If this is actually intended, I would strongly prefer to make this less "sneaky" but instead explicitly specify which L2/Linf error is returned from `analyze_solution`. If this was accidental, I think renaming the variables is sufficient, as I don't see the benefit of doing convergence analysis on anything but the conservative variables.

Reply:
You're right, thanks for catching that. It was accidental and fixed in the 2D version. It should also be fixed here.